lib/fix-google-html.js

```diff
@@ -552,6 +572,91 @@ function fixChecklists (node) {
   });
 }
 
+function getTextContent (node) {
+  if (node.type === 'text') {
+    return node.value;
+  }
+  else if (node.children) {
+    return node.children.map(getTextContent).join('');
+  }
+  else {
+    return '';
+  }
+}
+
+/**
+ * Create IDs for headings, and update internal links to use those IDs.
+ * @param {RehypeNode} node 
+ * @param {*} rawInternalContent 
+ */
+function fixInternalLinks (node, rawInternalContent) {
+
+  // get the internal headings
+  const internalContent = tryCatch(() => JSON.parse(JSON.parse(rawInternalContent).data).resolved)
```

Comment:
Could you move this parsing outside the function for headings? I think it's likely we'll want to use the data elsewhere in the future, so parsing shouldn't be buried in the place where the data is used. I think up-front in the convert function probably the right place for it, rather than in the `fix-google-html` module.